CondTools/SiPhase2Tracker/test/DTCCablingMapProducer_write.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("DTCCablingMapProducer")
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+# Load CondDB service
+process.load("CondCore.CondDB.CondDB_cfi")
+
+# output database (in this case local sqlite file)
+process.CondDB.connect = 'sqlite_file:OuterTrackerDTCCablingMap.db'
+
+# A data source must always be defined. We don't need it, so here's a dummy one.
+process.source = cms.Source("EmptyIOVSource",
+    timetype = cms.string('runnumber'),
+    firstValue = cms.uint64(1),
+    lastValue = cms.uint64(1),
+    interval = cms.uint64(1)
+)
+
+# We define the output service.
+process.PoolDBOutputService = cms.Service("PoolDBOutputService",
+    process.CondDB,
+    timetype = cms.untracked.string('runnumber'),
+    toPut = cms.VPSet(cms.PSet(
+        record = cms.string('TrackerDetToDTCELinkCablingMapRcd'),
+        tag = cms.string('DTCCablingMapProducerUserRun')
+    ))
+)
+
+process.otdtccablingmap_producer = cms.EDAnalyzer("DTCCablingMapProducer",
+    record = cms.string('TrackerDetToDTCELinkCablingMapRcd'),
+    generate_fake_valid_gbtlink_and_elinkid = cms.bool(True),
+    modulesToDTCCablingCSVFileNames = cms.vstring(
+      #"CondTools/SiPhase2Tracker/data/OT613_200_IT4025__ModulesToDTCsAllOuter.csv",
+      #"CondTools/SiPhase2Tracker/data/OT614_200_IT404_layer2_10G__ModulesToDTCsNegOuter.csv",
+      #"CondTools/SiPhase2Tracker/data/OT614_200_IT404_layer2_10G__ModulesToDTCsPosOuter.csv",
+      "CondTools/SiPhase2Tracker/data/OT614_200_IT404_layer2_10G__CMSSWCablingMapOuter.csv"
```

Comment:
was it decided if some of these files will eventually be put into cms-data?
running this now, I incur in the error:
```
----- Begin Fatal Exception 14-Mar-2019 22:34:29 CET-----------------------
An exception of category 'FileInPathError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=DTCCablingMapProducer label='otdtccablingmap_producer'
Exception Message:
edm::FileInPath unable to find file CondTools/SiPhase2Tracker/data/OT614_200_IT404_layer2_10G__CMSSWCablingMapOuter.csv anywhere in the search path.
The search path is defined by: CMSSW_SEARCH_PATH
${CMSSW_SEARCH_PATH} is: /tmp/musich/CMSSW_10_6_X_2019-03-14-1100/poison:/tmp/musich/CMSSW_10_6_X_2019-03-14-1100/src:/tmp/musich/CMSSW_10_6_X_2019-03-14-1100/external/slc7_amd64_gcc700/data:/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_X_2019-03-14-1100/src:/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_X_2019-03-14-1100/external/slc7_amd64_gcc700/data
Current directory is: /tmp/musich/CMSSW_10_6_X_2019-03-14-1100/src/CondTools/SiPhase2Tracker/test
----- End Fatal Exception -------------------------------------------------

```

Reply:
AFAIK there was no "decision" taken, maybe it's just best if I create a PR there and see what happens.

Reply:
Ahh, damn. I cannot create a "CondTools-SiPhase2Tracker" repository there. I'll need to wait @iahmad-khan to do so, so I can clone and issue a PR

Reply:
The repo is available now 
https://github.com/cms-data/CondTools-SiPhase2Tracker
put the file(s) there as PR

Reply:
@luigicalligaris @mmusich for the time being there is only one file in that repository. I will push it forward, but in case you need to update it to have the test working

Reply:
Let me cross-check the filenames

Reply:
Hi @fabiocos , I corrected the filename to match the file uploaded in cms-data